dm/master/election.go

```diff
@@ -126,11 +128,24 @@ func (s *Server) closeLeaderClient() {
 }
 
 func (s *Server) isLeaderAndNeedForward() (isLeader bool, needForward bool) {
+	retry := 10
+	wait := 100 * time.Millisecond
+	for ; retry > 0; retry-- {
+		if s.leader.Get() == oneselfStartingLeader {
+			time.Sleep(wait)
```

Comment:
if we want to wait more time, `time.After` (or tick) combine with `ctx` is always better than `time.Sleep` because we may need to support cancel this waiting. (IMO, `time.Sleep` is not a good choice in production code)